### PR TITLE
New Relic integration with k6 through New Relic StatsD

### PR DESCRIPTION
**Pull request by gspncr**
A guide to using New Relic as a k6 output including setup, configuration, and example dashboarding.

### CODE REVIEW

**Comment by mostafa** [comment]
Hi @gspncr,

thanks for the article. Please include the screenshots/graphics in the commit.

**Comment by gspncr** [comment]
thank you @mostafa - done!

**Comment by ppcano** [comment]
Thanks @gspncr for your contribution 🙏👏

We had the NewRelic integration in our TODO list, but we didn't know NewRelic supported `statsd`. 

It looks great. We are gonna test it to get familiar with the integration and we will merge this PR.

Note: we should not forget to mention the integration on the [Integrations page](https://k6.io/docs/integrations) and [Results output page](https://k6.io/docs/getting-started/results-output#output-plugins).

File: src/data/markdown/docs/01 guides/03 Results visualization/NewRelic.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+```sql
+SELECT sum(k6.http_req_duration.sum.percentiles) AS '90th' FROM Metric WHERE percentile = 90 TIMESERIES
+```
+</div>
+
+**Max, Average, Median Request Duration Duration**
+<div class="code-group" data-props='{"labels": [""]}'>
+
+```sql
+SELECT max(k6.http_req_duration) AS 'Max Duration', average(k6.http_req_duration.median) AS 'Median', average(k6.http_req_duration.mean) AS 'Avg' FROM Metric TIMESERIES
```
[the remark this answers is not in the document]
Comment (gspncr) [reply src/data/markdown/docs/01 guides/03 Results visualization/NewRelic.md]: thanks! resolved that - should have been calling max on the summary metric 😄

**Comment by simskij** [comment]
Thank you for your contribution! Merging 🥳  🎉  🙇‍♂️

**Comment by ppcano** [comment]
Published at https://k6.io/docs/results-visualization/new-relic

Thanks @gspncr

**Comment by gspncr** [comment]
🥳 Amazing - thank you @ppcano @simskij @mostafa looking forward to what more we can do in the future! 🚀

**Pull request merged**